OpenEdXMobile/src/main/java/org/edx/mobile/view/MainTabsDashboardFragment.java

```diff
@@ -103,6 +107,26 @@ public void onFragmentSelected() {
         return items;
     }
 
+    private void requestPostNotificationsPermission() {
+        permissionListener = new PermissionListener() {
+            @Override
+            public void onPermissionGranted(String[] permissions, int requestCode) {
+            }
+
+            @Override
+            public void onPermissionDenied(String[] permissions, int requestCode) {
+            }
+        };
```

Comment:
this block of code seems useless here, so better to remove it.

OR wrap with the condition below.